common/religion/religions/shathgral_group.txt

```diff
@@ -267,7 +293,14 @@
 		yshaarj_worship = {
 			#Main Group
 			color = rgb { 40 40 5 }
+			religious_head = d_yshaarj_worship
 			icon = custom_faith_1
+			
+			holy_site = ahnqiraj
+			holy_site = zulnazman
+			holy_site = ulduar
+			holy_site = storm_isle
+			holy_site = lulkail
```

Comment:
I think it will be fine if Y'Shaarj will have more holy sites (7 sha locations), just like vanilla have 7 holy sites for some religions; Instead of what we have now - the same 5 sites for every shathgral faith

Reply:
These locations in WoW aren't tied to important sites. Usually, it's just a place where someone got pissed off and summoned a big sha.

Changed holy sites in 65ebb00e59743b59df66b529d72661a0d6133aa1.